examples/action_rule_profile_additional_communities/main.tf

```diff
@@ -0,0 +1,31 @@
+terraform {
+  required_providers {
+    aci = {
+      source = "ciscodevnet/aci"
+    }
+  }
+}
+
+provider "aci" {
+  username = ""
+  password = ""
+  url      = ""
+  insecure = true
+}
+
+resource "aci_tenant" "foo_tenant" {
+  name        = "tf_tenant"
+  description = "tenant created while acceptance testing"
+}
+
+resource "aci_action_rule_profile" "foo_action_rule_profile" {
+  name        = "tf_rule"
+  description = "action_rule_profile created while acceptance testing"
+  tenant_dn   = aci_tenant.foo_tenant.id
+}
+
+resource "aci_action_rule_profile_additional_communities" "foo_rtctrl_set_add_comm" {
+  community              = "no-advertise"
```

Comment:
put the example to the same as proposed in the UI.

Reply:
UI does not have the placeholder value and "no-advertise" is one of the options in UI.